src/gui/optionsdialog.h

```diff
@@ -174,6 +174,7 @@ private slots:
     bool setSslCertificate(const QByteArray &cert);
     bool schedTimesOk();
     bool webUIAuthenticationOk();
+    bool alternativeWebUIOk();
```

Comment:
* Maybe we should dervive from the old naming style to something like: 
`isAlternativeWebUIPathValid`?
* Add const: `bool isAlternativeWebUIPathValid() const;`

Reply:
Sure, this naming is a bit more clear.